tlf/test_common.go

```diff
@@ -16,6 +16,17 @@ func FakeID(b byte, public bool) ID {
 	return ID{bytes}
 }
 
+// FakeIDRandomOrBust creates a fake public or private TLF ID from the given
+// byte, and fill the rest with empty bytes.
```

Comment:
Right I'm not saying we need to change the `libkbfs` tests; just the server ones.